cuebot/src/main/java/com/imageworks/spcue/util/FrameSet.java

```diff
@@ -44,6 +48,140 @@ public int index(int idx) {
         return frameList.indexOf(idx);
     }
 
+    /**
+     * Return a string representation of a subset of a frame range.
+     *
+     * This approach was adapted from https://pypi.org/project/Fileseq/
+     * @param startFrame Start frame
+     * @param endFrame   End frame
+     * @param stride     The step between frames
+     * @param zFill      Left pad each frame ID with zeroes to make it this width.
+     * @return           String representation of the frame range, e.g. 1-1001x3
+     */
+    private String buildFrangePart(int startFrame, int endFrame, int stride, int zFill) {
```

Comment:
nit: Put all the private methods below the public methods, so future devs will have an easier time quickly seeing the various public methods they might want to use.